cmd/gapit/memory.go

```diff
@@ -95,35 +100,37 @@ func (verb *memoryVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		}
 	}
 
-	fmt.Printf("%v memory allocations\n", len(mem.Allocations))
+	w := tabwriter.NewWriter(os.Stdout, 4, 4, 0, ' ', 0)
+	fmt.Fprintf(w, "%v memory allocations\n", len(mem.Allocations))
 	sort.Slice(mem.Allocations, func(i, j int) bool {
 		return mem.Allocations[i].Handle < mem.Allocations[j].Handle
 	})
+
 	for _, alloc := range mem.Allocations {
-		fmt.Println("Name:", alloc.Name)
-		fmt.Println("\tDevice:      ", alloc.Device)
-		fmt.Println("\tMemory Type: ", alloc.MemoryType)
-		fmt.Println("\tSize:        ", alloc.Size)
+		fmt.Fprintln(w, "Name:", alloc.Name)
+		fmt.Fprintln(w, "\tDevice:      ", alloc.Device)
```

Comment:
The main reason I suggested the tab writer was that it allows you to align the fields here without having to have a load of whitespace indentations.